tests/app/dao/test_fact_notification_status_dao.py

```diff
@@ -267,6 +269,56 @@ def test_fetch_notification_status_for_service_for_today_and_7_previous_days(not
     assert results[3].count == 19
 
 
+@freeze_time('2018-10-31T18:00:00')
+def test_fetch_notification_status_for_service_for_today_and_7_previous_days_for_high_volume_service(
+        notify_api, notify_db_session
+):
+    service_1 = create_service(service_name='service_1')
+    sms_template = create_template(service=service_1, template_type=SMS_TYPE)
+    sms_template_2 = create_template(service=service_1, template_type=SMS_TYPE)
+    email_template = create_template(service=service_1, template_type=EMAIL_TYPE)
+
+    create_ft_notification_status(date(2018, 10, 29), 'sms', service_1, count=10)
+    create_ft_notification_status(date(2018, 10, 24), 'sms', service_1, count=8)
+    create_ft_notification_status(date(2018, 10, 29), 'sms', service_1, notification_status='created')
+    create_ft_notification_status(date(2018, 10, 29), 'email', service_1, count=3)
+    create_ft_notification_status(date(2018, 10, 26), 'letter', service_1, count=5)
+
+    create_notification(sms_template, created_at=datetime(2018, 10, 31, 11, 0, 0))
+    create_notification(sms_template_2, created_at=datetime(2018, 10, 31, 11, 0, 0))
+    create_notification(sms_template, created_at=datetime(2018, 10, 31, 12, 0, 0), status='delivered')
+    create_notification(email_template, created_at=datetime(2018, 10, 31, 13, 0, 0), status='delivered')
```

Comment:
Optional: Would appreciate a comment above these to say these were created today and are not found in the results